Tasks/VsTest/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -40,6 +40,7 @@
   "loc.input.help.configuration": "Configuration against which the tests should be reported. If you have defined a variable for configuration in your build task, use that here.",
   "loc.input.label.publishRunAttachments": "Upload Test Attachments",
   "loc.input.help.publishRunAttachments": "Opt in/out of publishing test run level attachments.",
+  "loc.messages.PathDoesNotExist": "%s path does not exist.",
```

Comment:
If access is denied for a file, the call to check isFile fails throwing an error 'Error: EPERM: operation not permitted', in which case the task will exit and the above message will not be printed.